### ISSUE TITLE
Again problems with CSOM SendEmail

### ISSUE THREAD

**Issue by cwdata** [SharePoint/sp-dev-docs#6291]
## Category

- [x] Question

## Question

We are again seeing problems when using
`Microsoft.SharePoint.Client.Utilities.Utility.SendEmail(context, properties);`
through CSOM.
Most of the time mails sent this way get delivered, sometimes they don't. We are always sending mails to recipients in the same tenant.
In the past this method threw errors (see #4929) but since several weeks no more errors are thrown. Sending the mails just fails silently so that our retry logic doesn't get triggered.
Yesterday I sent 5 mails with nearly the same content in a 2 minute timeframe to the same recipient. 3 got delivered, 2 are still missing.
Most likely this is not a problem in the SendEmail method but again some type of misconfiguration in the underlying mail server infrastructure. But how could we report that to Microsoft? Support just tells us: This is coming from code so it's a developer issue.

## Environment details (if applicable)

- **Your Developer Environment**: Windows 10
- **Target Environment**: SharePoint Online
- **Framework**: .NET Framework v4.7
- **Browser(s)**: All
- **Tooling**: Visual Studio 2019

**Comment by Siglid** [comment]
@cwdata Did you ever resolve this issue? I have encountered it myself recently and having issues in finding a way to resolve

**Comment by cwdata** [comment]
@Siglid,
no, unfortunately there's nothing we can do from our end to resolve this. We switched to Graph for sending mails, which incurs a lot of additional configuration. But beware, Graph is also not a guarantee that your mails get delivered. We encountered the exact same issue there too, but only for a couple of days in several months. I think, if more and more people will go that route we'll see things getting worse on the Graph side too.
